spynnaker/pyNN/models/abstract_models/abstract_data_specable_vertex.py

```diff
@@ -15,7 +15,10 @@
 
 import tempfile
 import os
+import threading
 
+#used to stop file conflicts
```

Comment:
[PEP8:Block Comments](https://www.python.org/dev/peps/pep-0008/#block-comments) - should have a space after the #